examples/radio.py

```diff
@@ -0,0 +1,30 @@
+# A micro:bit Firefly.
+# By Nicholas H.Tollervey. Released to the public domain.
+import radio
+from microbit import display, Image, button_a, sleep
+import random
+
+# Create the "flash" animation frames.
+flash = [Image((str(i) * 5 + ':') * 5) for i in range(9, -1, -1)]
```

Comment:
I think you win the most obscure one-liner of the year award with that one... :-) I hadn't thought of using invert. However, the result of the list comprehension is in reverse order so the convoluted `range` call should remain (or we do [reversed([...])] which seems even more obscure) ;-)